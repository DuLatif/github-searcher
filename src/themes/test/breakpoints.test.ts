import { describe, expect, test } from "vitest";
import { readFileSync } from "fs";

describe("Theme Breakpoints", () => {
  test("should match scss breakpoints variable snapshot", () => {
    expect(readFileSync(__dirname + "./../breakpoints.scss")).toMatchSnapshot();
  });
});
