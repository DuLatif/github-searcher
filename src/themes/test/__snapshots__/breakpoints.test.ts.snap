// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Theme Breakpoints > should match scss breakpoints variable snapshot 1`] = `
{
  "data": [
    36,
    120,
    115,
    58,
    32,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    115,
    109,
    58,
    32,
    54,
    48,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    109,
    100,
    58,
    32,
    57,
    48,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    108,
    103,
    58,
    32,
    49,
    50,
    48,
    48,
    112,
    120,
    59,
    13,
    10,
    36,
    120,
    108,
    58,
    32,
    49,
    53,
    51,
    54,
    112,
    120,
    59,
    13,
    10,
  ],
  "type": "Buffer",
}
`;
